### ISSUE TITLE
Grid scans should move detector in and open shutter

### ISSUE THREAD

**Issue by DominicOram** [DiamondLightSource/hyperion#878]
Currently detector motion is done GDA for grid scans. We should do this in Hyperion and wait on it properly

## Acceptance Criteria
* As early as possible Hyperion starts moving the detector and opening the shutter
* Just before data collection we wait on the movement to finish
* Spin off a new ticket to think about smaller plans still being callable on their own
